test/ILLink.RoslynAnalyzer.Tests/Verifiers/CSharpAnalyzerVerifier`1.cs

```diff
@@ -40,7 +40,7 @@ public static DiagnosticResult Diagnostic (DiagnosticDescriptor descriptor)
 			=> CreateCompilation (CSharpSyntaxTree.ParseText (src), globalAnalyzerOptions, additionalReferences);
 
 		public static async Task<Compilation> GetCompilation (string source, IEnumerable<MetadataReference>? additionalReferences = null)
-			=> (await CSharpAnalyzerVerifier<RequiresAssemblyFilesAnalyzer>.CreateCompilation (source, additionalReferences: additionalReferences ?? Array.Empty<MetadataReference> ())).Compilation.Compilation;
+			=> (await CSharpAnalyzerVerifier<TAnalyzer>.CreateCompilation (source, additionalReferences: additionalReferences ?? Array.Empty<MetadataReference> ())).Compilation.Compilation;
```

Comment:
I think the fact that `TAnalyzer` is completely meaningless here means we should probably move a base `CreateCompilation` helper out of this class and into some neutral test file. There are no analyzers on a base compilation, so taking in type parameters is just confusing.